generator/src/lib.rs

```diff
@@ -1763,56 +1784,18 @@ pub fn derive_setters(
         .map(|extends| format_ident!("Extends{}", name_to_tokens(extends)))
         .map(|extends| {
             quote! {
-                unsafe impl #extends for #name_builder<'_> {}
-                unsafe impl #extends for #name {}
+                unsafe impl #lifetime #extends for #name #lifetime {}
```

Comment:
Couldn't this stay as-is or am I not knowing something about lifetimes in trait impls?


```suggestion
                unsafe impl #extends for #name<'_> {}
```